test/unit/BondingManager.js

```diff
@@ -422,28 +497,97 @@ contract("BondingManager", accounts => {
                     assert.equal((await bondingManager.getDelegator(delegator))[2], transcoder1, "wrong delegateAddress")
                 })
 
-                describe("new delegate is registered transcoder", () => {
-                    it("should update new delegate's position in transcoder pool", async () => {
-                        await bondingManager.bond(0, transcoder1, {from: delegator})
+                describe("old delegate is registered transcoder", () => {
+                    describe("new delegate is a registered transcoder", () => {
+                        it("should update new delegate's position in transcoder pool", async () => {
+                            await bondingManager.bond(0, transcoder1, {from: delegator})
+                            await fixture.roundsManager.setMockUint256(functionSig("currentRound()"), currentRound + 1)
+                            // New delegate was not previously first transcoder in pool and now is
```

Comment:
Could you please elaborate? Since we're bonding an extra 2000 from `delegator` to `transcoder0` in the `beforeEach` call, the comment should be correct I think. 

```
Before test 

1. transcoder0 - 1000 self, 3000 from delegator
2. transcoder1 - 2000 self

After test

1. transcoder1 - 2000 self, 3000 from delegator
2. transcoder0 - 1000 self
```